frontend/src/components/ElectionForm/ElectionForm.tsx

```diff
@@ -130,26 +264,401 @@ const ElectionForm = ({ authSession, onSubmitElection, prevElectionData, submitT
-                        }
-                    </Grid>
+                    <Fade in={page === 'ElectionDetails'} mountOnEnter unmountOnExit timeout={{ appear: 500, enter: 500, exit: 0 }}>
+                        <div>
+                            <ElectionDetails election={election} applyElectionUpdate={applyElectionUpdate} getStyle={getStyle} onBack={() => setPage('ElectionDetails')} onNext={() => setPage('RaceDetails')} />
+                        </div>
+                    </Fade>
+                    <Fade in={page === 'RaceDetails'} mountOnEnter unmountOnExit timeout={{ appear: 500, enter: 500, exit: 0 }}>
+                        <div>
+                            <Races election={election} applyElectionUpdate={applyElectionUpdate} getStyle={getStyle} onBack={() => setPage('ElectionDetails')} onNext={() => setPage('Open?')} />
+                        </div>
+                    </Fade>
+                    <Question
+                        Enable={page === 'Open?'}
+                        Question='Will this election be open to anyone?'
+                        HelpText=''
+                        Option1={{
+                            Answer: 'Yes',
+                            HelpText: 'For quick polls or demos',
+                            GotoPage: 'Limit?',
+                            election: election,
+                            setPage: setPage,
+                        }}
+                        Option2={{
+                            Answer: 'No',
+                            HelpText: 'For when you want some restrictions on who can vote',
+                            GotoPage: 'VoterList?',
+                            election: election,
+                            setPage: setPage,
+                        }}
+                        BackPage='RaceDetails'
+                        setPage={setPage}
+                    />
+                    <Question
+                        Enable={page === 'Limit?'}
+                        Question='Do you want to limit to one vote per person?'
+                        HelpText=''
+                        Option1={{
+                            Answer: 'Yes',
+                            HelpText: 'For most polls',
+                            GotoPage: 'Scenario1',
+                            election: election,
+                            setPage: setPage,
+                        }}
+                        Option2={{
+                            Answer: 'No',
+                            HelpText: 'For when you want to vote multiple times on one device, demos, tabling, etc.',
+                            GotoPage: 'Scenario2',
+                            election: election,
+                            setPage: setPage,
+                        }}
+                        BackPage='Open?'
```

Comment:
nit

I would be nice if the BackPage could be automatically determined based on the previous GotoPage. I'm not sure how easy this change to would be to make, but it would make this page easier to maintain in the future

Reply:
Yea I struggled with this. There's a few ways to potentially fix this. We could put the page into the url and use the built in navigation but that might break things. Or we could keep an array of the page history and manage history ourselves? For a future PR.